Libraries/Animated/src/NativeAnimatedHelper.js

```diff
@@ -153,6 +153,8 @@ const STYLES_WHITELIST = {
   borderTopLeftRadius: true,
   borderTopRightRadius: true,
   borderTopStartRadius: true,
+  width: true,
+  height: true,
```

Comment:
Might want to get rid of this whitelist, right now we support all number based props.